### PR TITLE
README updates

### LINKED ISSUE #2376: [QST] ArangoDB - cuGraph
Hi there, [ArangoDB](https://github.com/arangodb/arangodb) is currently maintaining a cuGraph adapter that allows developers to convert their ArangoDB graphs to cuGraph, and vice versa.

We are curious to know whether this would be worth listing somewhere as a cuGraph-powered project, or even as a blog post.

Would this be of interest?

https://github.com/arangoml/cugraph-adapter

### PR DESCRIPTION
**Pull request by BradReesWork**
updated the README

closes #2376

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -210,6 +211,14 @@ Please see our [guide for contributing to cuGraph](CONTRIBUTING.md).
...
 Python API documentation can be generated from [docs](docs) directory.
 
+------
+# Projects that use cuGraph
+
+(alphabetical order)
+* ArangoDB - a free and open-source native multi-model database system  - https://www.arangodb.com/
+* CuPy - "NumPy/SciPy-compatible Array Library for GPU-accelerated Computing with Python" -  https://cupy.dev/
+* Memgraph - In-memory database - https://memgraph.com/
+* ScanPy - Scanpy is a scalable toolkit for analyzing single-cell gene expression data - https://scanpy.readthedocs.io/en/stable/
```
Comment (rlratzel) [inline README.md:221]: minor nit just for consistency with ArangoDB
```suggestion
* ScanPy - a scalable toolkit for analyzing single-cell gene expression data - https://scanpy.readthedocs.io/en/stable/

```

File: README.md [hunk 2]
```diff
@@ -2,7 +2,9 @@
...
+The [RAPIDS](https://rapids.ai) cuGraph library is a collection of GPU accelerated graph algorithms that process data found in [GPU DataFrames](https://github.com/rapidsai/cudf).  The vision of cuGraph is _to make graph analysis ubiquitous to the point that users just think in terms of analysis and not technologies or frameworks_.  To realize that vision, cuGraph operates, at the Python layer, on GPU DataFrames, thereby allowing for seamless passing of data between ETL tasks in [cuDF](https://github.com/rapidsai/cudf) and machine learning tasks in [cuML](https://github.com/rapidsai/cuml).  Data scientists familiar with Python will quickly pick up how cuGraph integrates with the Pandas-like API of cuDF.  Likewise, users familiar with NetworkX will quickly recognize the NetworkX-like API provided in cuGraph, with the goal to allow existing code to be ported with minimal effort into RAPIDS.
```
Comment (rlratzel) [inline README.md:5]: This sentence might read better with fewer commas, maybe like this:
```
To realize that vision, cuGraph operates at the Python layer on GPU DataFrames, thereby allowing for seamless passing of data between ETL...
```

File: README.md [hunk 3]
```diff
@@ -2,7 +2,9 @@
...
+
+As you move down the software stack, graph structure and processing becomes less obfuscated and understanding of graph theory is needed.  We provide a Python layer, called pylibcugraph, for users wishing to integrate with cuGraph at the Python layer.  For users familiar with C/C++/CUDA and graph structures, can access libcugraph and libcugraph_c for low level integration.
```
Comment (rlratzel) [inline README.md:7]:
```suggestion
While the high-level cugraph python API provides an easy-to-use and familiar interface for data scientists that's consistent with other RAPIDS libraries in their workflow, some use cases require access to lower-level graph theory concepts.  For these users, we provide an additional Python API called pylibcugraph, for applications that require a tighter integration with cuGraph at the Python layer.  Users familiar with C/C++/CUDA and graph structures can access libcugraph and libcugraph_c for low level integration outside of python.
```

**Review by rlratzel — COMMENTED** [review]
Looks good, I just had a few suggestions:

**Review by ChuckHastings — APPROVED** [review]

**Comment by BradReesWork** [comment]
@gpucibot merge

**Pull request merged**